### PR TITLE
update supported browsers and add a few more

### PR DESCRIPTION
**Pull request by silverwind**
Now includes Firefox ESR, Opera and a few more browser with usages between 0.5% and 1%. defaults resolves to > 0.5%, last 2 versions, Firefox ESR, not dead.

#### before
and_chr 78
and_uc 12.12
chrome 79
chrome 78
edge 18
firefox 71
firefox 70
ie 11
ios_saf 13.3
ios_saf 13.2
ios_saf 13.0-13.1
ios_saf 12.2-12.4
op_mini all
safari 13
safari 12.1
samsung 10.1

#### after
and_chr 78
and_ff 68
and_qq 1.2
and_uc 12.12
android 76
baidu 7.12
chrome 79
chrome 78
chrome 77
chrome 49
edge 18
edge 17
firefox 71
firefox 70
firefox 68
ie 11
ie_mob 11
ios_saf 13.3
ios_saf 13.2
ios_saf 13.0-13.1
ios_saf 12.2-12.4
kaios 2.5
op_mini all
op_mob 46
opera 64
opera 63
safari 13
safari 12.1
safari 5.1
samsung 10.1
samsung 9.2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,9 +28,7 @@
     "webpack-cli": "3.3.10"
   },
   "browserslist": [
-    "> 1%",
-    "last 2 firefox versions",
-    "last 2 safari versions",
+    "defaults",
     "ie 11"
```
Comment (6543) [inline package.json]: who is checkinf ig IE still works :sweat_smile: ?

**Review by sapk — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (silverwind) [reply package.json]: IE 11 is already partially broken in some regards, and I don't think we got any recent bug reports about it. I've been meaning to remove its support a while ago, maybe we should do it now.

Comment (sapk) [reply package.json]: ~What does it cost to keep it ?~ we still have edge setup it should be enough.

Comment (davidsvantesson) [reply package.json]: I've used it sometime when needing an extra session in addition to Chrome 😛.

Comment (silverwind) [reply package.json]: I guess I'll do it. Windows 7 EOL is in 2 days and Windows 10 users have Edge installed as an alternative.

Better to make a clear statement about it not being supported than have half-assed support as we do right now with various stuff broken (swagger-ui being one of them).

**Review by 6543 — APPROVED** [review]

**Comment by silverwind** [comment]
IE 11 removed as per discussion. I had to explicitely remove it because it still came in the `defaults`.

Also removed Safari 5.1. I think this is the long dead Windows version of Safari from 2011 which I would not expect to work.

**Comment by tboerger** [comment]
Is this selecting based on any official average statistics? I would prefer some standard statistics before dropping support for browsers in minor version changes.

**Comment by sapk** [comment]
Maybe just stay to `defaults` even if it contain IE11 ?

**Comment by silverwind** [comment]
Browser user data comes from [caniuse](https://caniuse.com) data which I think is sourced from [statcounter](http://gs.statcounter.com/).

**Comment by silverwind** [comment]
> Maybe just stay to defaults even if it contain IE11 ?

I wouldn't. Either we support it or not. It will block certain features like CSS variable so I think it's time we do it.

By the way GitHub dropped IE 11 support in July 2018, I say it's long overdue.

**Comment by silverwind** [comment]
Decided to give IE11 a try and with https://github.com/go-gitea/gitea/pull/9735 applied, it actually does not look so bad, no JS errors while browsing a few pages.

Based on that, I think `defaults` is good enough for now, so IE support lives on.

It does not noticeable increase build size (only about 1kB more CSS, JS stays the same).

File: package.json [hunk 2]
```diff
@@ -28,9 +28,6 @@
     "webpack-cli": "3.3.10"
   },
   "browserslist": [
-    "> 1%",
-    "last 2 firefox versions",
-    "last 2 safari versions",
-    "ie 11"
+    "defaults"
```
Comment (bagasme) [inline package.json:34]: So what `defaults` then?

Comment (silverwind) [reply package.json:34]: See first post. It is `0.5%, last 2 versions, Firefox ESR, not dead` which resolves to those browsers listed there.

**Review by sapk — APPROVED** [review]

**Pull request merged**